README.md

```diff
@@ -1 +1,147 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello Microverse
+
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+      - [Tech Stack](#tech-stack)
+      - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+This is the first project for sitting up all the tools required for working remotely
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- This repository is just for the purpose of to checking all the required tools for working projects remotely.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [OPTIONAL] Because of the missing tag `>` in this paragraph, it messes up the layout of the `Live Demo` header, please ensure adding the closing symbol  👍🏼